pkg/client/unversioned/testclient/fake_configdatas.go

```diff
@@ -0,0 +1,92 @@
+/*
+Copyright 2015 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package testclient
+
+import (
+	"k8s.io/kubernetes/pkg/api"
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/fields"
+	"k8s.io/kubernetes/pkg/labels"
+	"k8s.io/kubernetes/pkg/watch"
+)
+
+// Fake implements ConfigDataInterface. Meant to be embedded into a struct to get a default
+// implementation. This makes faking out just the method you want to test easier.
+type FakeConfigDatas struct {
+	Fake      *Fake
+	Namespace string
+}
+
+func (c *FakeConfigDatas) Get(name string) (*extensions.ConfigData, error) {
+	obj, err := c.Fake.Invokes(NewGetAction("configdatas", c.Namespace, name), &extensions.ConfigData{})
```

Comment:
Actually, it looks like resource name is wrong throughout this file. `configDatas` matches what's in the client implementation.